sdk/core/Azure.Core/src/Shared/IOperationPollingStrategy.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Core
+{
+    /// <summary>
+    /// Polling strategy of <see cref="OperationInternalBase"/>.
+    /// </summary>
+    internal interface IOperationPollingStrategy
+    {
+        /// <summary>
+        /// Gets interval for current iteration of polling.
+        /// </summary>
+        /// <remarks>Note that the value could change per call.</remarks>
+        public TimeSpan PollingInterval { get; }
```

Comment:
I think this should be a method named GetNextWait or something like that since it will be different every time in a lot of cases.

Reply:
done